crates/kubelet/src/volume/persistentvolumeclaim.rs

```diff
@@ -210,7 +210,7 @@ pub(crate) async fn populate(
     // TODO(bacongobbler): implement node_unstage_volume(). We'll need to
     // persist the staging_path somewhere so we can recall that information
     // during unpopulate()
-    let staging_path = TempDir::new(&csi.volume_handle)?;
+    let staging_path = Builder::new().prefix("&csi.volume_handle").tempdir()?;
```

Comment:
We should probably be using `tokio::fs` here. That way we're not blocking the thread. Everything else is fine - mio_uds_windows is third-party code.

Also why are we quoting the literal string `&csi.volume_handle` here? Shouldn't the temporary directory use the literal volume handle name as a prefix instead?

Reply:
We absolutely should not be quoting that, you are right. The danger of performing multiple **almost** identical changes in short order.

I'm afraid I don't fully understand your tokio::fs comment though. Afaik that offers no temp dir functionality, are you suggesting to create and cleanup a temp dir "manually" ?